features/configuration/filter_sensitive_data.feature

```diff
@@ -24,6 +24,11 @@ Feature: Filter sensitive data
   substitution string so that the interaction will be identical to what was
   originally recorded.
 
+  Warning, the data you're filtering may have been escaped or encoded.
+  Thus you may need to escape the sensitive text. For example, if
+  "some&password" is in your cassette, you will need to filter
+  "some&amp;password". You can also use `CGI::escapeHTML('some&password')`.
```

Comment:
Great idea and really helpful (I was stuck with this for a while).

It might be good to add that query params need to use `CGI::escape` rather than `CGI::escapeHTML`

Reply:
@timdiggins Cool, do you have a wording example, or would you like to make a follow-up PR?

Reply:
Good point @olleolleolle. Here's a propsal. Looks like this branch is now out of date, so may make sense for @hirowatari to do all the changes, if agreeable. I can do a follow-up PR if needed.

    Warning, the data you're filtering may have been escaped or encoded.
    Thus you may need to escape the sensitive text. For example, if 
    "some&password" is in your cassette within a context which is html 
    escaped, you will need to filter "some&amp;password" or 
    `CGI::escapeHTML('some&password')`. Similarly if it's escaped in a 
     query string, then you will need to filter "some%26password" 
     or `CGI::escape('some&password')`

Reply:
PS could leave out the last sentence if the text is now too long, it's more the "which is html escaped" that helps indicate that escapeHTML is only one of the possible escaping solutions